src/r_string.rs

```diff
@@ -900,6 +900,24 @@ impl RString {
             .map_err(|e| Error::new(exception::encoding_error(), format!("{}", e)))
     }
 
+    /// Returns `self` as an owned Rust `Bytes`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use magnus::RString;
+    /// use bytes::Bytes;
+    /// # let _cleanup = unsafe { magnus::embed::init() };
+    ///
+    /// let s = RString::new("example");
+    /// assert_eq!(s.to_bytes(), Bytes::from("example"));
+    /// ```
+    #[cfg(feature = "bytes-crate")]
+    pub fn to_bytes(self) -> bytes::Bytes {
```

Comment:
If you add `#[cfg_attr(docsrs, doc(cfg(feature = "bytes-crate")))]` here the docs on docs.rs will call out that this is only available with the `bytes-crate` feature.

If you want to see what this looks like locally you can approximate the docs.rs build with:
``` sh
env RUSTDOCFLAGS='--cfg docsrs' cargo +nightly doc --all-features
```